lib/parzival/tools.ex

```diff
@@ -133,6 +141,12 @@ defmodule Parzival.Tools do
   """
   def get_announcement!(id), do: Repo.get!(Announcement, id)
 
+  def get_announcement!(id, opts \\ []) do
```

Comment:
This function doesn't work properly.
It should be like this:
```
def get_requirement!(id, preloads \\ []),
    do: Repo.get!(Requirement, id) |> Repo.preload(preloads)
```

Reply:
Thanks for the feedback :pray: